Task 1/Assets/Scripts/MeshCalculator.cs

```diff
@@ -0,0 +1,134 @@
+using UnityEngine;
+using System.Collections.Generic;
+
+public class MeshCalculator
+{
+    public Vector3 sizes;
+
+    private float xHalf;
+    private float yHalf;
+    private float zHalf;
+
+    public List<Vector3> vertices = new List<Vector3>();
+    public List<int> triangles = new List<int>();
+
+    private void CalculateCoordinates()
+    {
+        xHalf = sizes.x / 2;
+        yHalf = sizes.y / 2;
+        zHalf = sizes.z / 2;
+    }
+
+    private void GenerateFront()
+    {
+        vertices.Add(new Vector3(-1 * xHalf, -1 * yHalf, -1 * zHalf));//0
```

Comment:
https://docs.unity3d.com/ScriptReference/Vector3.Scale.html

Reply:
Please read down all classes api and remember each and EVERY method:
Time, Vector2/3/4, Quaternion, Tranform, Mathf

Reply:
https://docs.unity3d.com/Manual/ScriptingImportantClasses.html